Assets/_Laz/Scripts/UserInterface/DebugBehaviour/DebugKeyHandler.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine.InputSystem;
+
+namespace Laz
+{
+    public partial class DebugUIBehaviour
+    {
+        private int GetDebugBuildIndex()
```

Comment:
Nice, it at least looks more organized.

Reply:
I was thinking in passing, what if you had a struct where it contains the Key + buildIndex

struct KeyToBuildIndex {
    Key key;
   int buildIndex
}

and then 

Foreach(var struct in ListOfStruct) {
     if (keyboard.current[struct.key].wasPressed)
           buildIndex = struct.buildIndex
}


Something like this.

Reply:
Wouldn't a dictionary work just as well then??

Reply:
sure, what ever structure u wanna use